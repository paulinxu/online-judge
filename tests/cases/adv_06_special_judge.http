###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 99.0); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 429
# content-type: application/json
# date: Thu, 25 Jul 2024 15:32:37 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"The error is within bounds","result":"Accepted","time":0}],"created_time":"2024-07-25T15:32:37.671356997Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 99.0); }","user_id":0},"updated_time":"2024-07-25T15:32:37.859043304Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 80.0); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 435
# content-type: application/json
# date: Thu, 25 Jul 2024 15:32:37 GMT
# 
# {"cases":[{"id":0,"info":"","result":"Compilation Success","time":0},{"id":1,"info":"The error is beyond bounds","result":"Wrong Answer","time":0}],"created_time":"2024-07-25T15:32:37.860802746Z","id":1,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 80.0); }","user_id":0},"updated_time":"2024-07-25T15:32:38.046480880Z"}
