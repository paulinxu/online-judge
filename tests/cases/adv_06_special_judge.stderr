[2024-07-25T15:32:36Z INFO  actix_server::builder] starting 16 workers
[2024-07-25T15:32:36Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2024-07-25T15:32:37Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000241
[2024-07-25T15:32:37Z INFO  oj::jobs_module::spj] ["./tests/data/guessing/judge.py", "TMPDIR/test.out", "./tests/data/guessing/1.ans"]
[2024-07-25T15:32:37Z INFO  oj::jobs_module::spj] Code ran successfully
[2024-07-25T15:32:37Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 429 "-" "-" 0.188240
[2024-07-25T15:32:38Z INFO  oj::jobs_module::spj] ["./tests/data/guessing/judge.py", "TMPDIR/test.out", "./tests/data/guessing/1.ans"]
[2024-07-25T15:32:38Z INFO  oj::jobs_module::spj] Code ran successfully
[2024-07-25T15:32:38Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 435 "-" "-" 0.185966
